vm/src/protocol/number.rs

```diff
@@ -120,7 +120,7 @@ pub struct PyNumberMethods {
     pub negative: Option<PyNumberUnaryFunc>,
     pub positive: Option<PyNumberUnaryFunc>,
     pub absolute: Option<PyNumberUnaryFunc>,
-    pub boolean: Option<PyNumberUnaryFunc<bool>>,
+    pub boolean: Option<PyNumberUnaryFunc<bool>>, // __bool__
```

Comment:
this is unrelated change.

```suggestion
    pub boolean: Option<PyNumberUnaryFunc<bool>>,
```